parser.go

```diff
@@ -1488,27 +1486,70 @@ func (p *parser) parseSchemaPropertiesFromStructFields(pkgPath, pkgName string,
 		isSliceOrMap := strings.HasPrefix(typeAsString, "[]") || strings.HasPrefix(typeAsString, "map[]")
 		isInterface := strings.HasPrefix(typeAsString, "interface{}")
 		if isSliceOrMap || isInterface || typeAsString == "time.Time" {
-			var err error
-			fieldSchema, err = p.parseSchemaObject(pkgPath, pkgName, typeAsString, true)
-			if err != nil {
-				p.debug(err)
-				return
+			splitType := strings.Split(typeAsString, "]")
```

Comment:
This logic is added so that we can parse nested arrays/slices. It was not correctly handling them and in cases would assign it was `interface{}`